hack/init-environment.sh

```diff
@@ -144,10 +146,30 @@ function install_flux () {
     -v ${env_dir}/.helm/cache:/root/.cache/helm
     ${HELM}
   )
+    echo ${helm[@]}
+}
+
+
+#####
+## Set up Flux CD's Helm Operator to manage deployments in GKE.
+#####
+function install_flux () {
+  local -r kubeconfig=$1 env_dir=$2
+
+  # Install the CRD definitions separately. Helm doesn't have
+  # a coherent story for handling these yet (since updating them
+  # the wrong way can result in all existing objects being deleted),
+  # so they're easier to handle out-of-band.
+  declare -ra kubernetes=$(configure_kubernetes ${kubeconfig})
```

Comment:
```suggestion
  declare -ra kubernetes=($(configure_kubernetes ${kubeconfig}))
```
I _think_ this is required to make it actually be an array (right now I think it's evaluating to an array of a single element)